data/gravity-model/prem.txt

```diff
@@ -0,0 +1,50 @@
+# A gravity profile that follows the description of
+# the preliminary reference Earth model (PREM) by
+# Dziewonski & Anderson (1981)
+# POINTS: 44
```

Comment:
Is there any case where you would ever need a higher resolution compared to what you have here? As this includes the whole Earth, you have ~25 points for the mantle, so do you think that covers the jumps well enough? Or is the gravity of PREM stepwise linear anyway?